### DISCUSSION TITLE
How to I successfully apply a bg color on a table row?

### DISCUSSION THREAD

**Discussion by Mashiane** [saadeghi/daisyui#1762] [Q&A] [answered]
Hi

I am trying to apply a bg-red to a 'tr', no avail.
If I apply a text color, eg text-white, it works and not for bg.

What am I missing?

PS: I also want to

**Comment by saadeghi** [comment, marked as answer]
Currently the cell (`<td>`) is having the background color so you can override the color by adding `bg-*` class to each cells.  
Or you can use a class like `[&>*]:bg-blue-200` on `<tr>` which applies bg color to all items inside `<tr>`.  
(This is going to get fixed in daisyUI 3.0)

Here's an example: https://play.tailwindcss.com/ShS97eXjqK
